.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Some stuff I often put into my settings.json:
```
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  },
  "editor.defaultFormatter": "esbenp.prettier-vscode", 
```
The default formatter makes sure that the formatting gets done with the same extension even if someone else uses a different formatter for theirs

Reply:
Correct, the only notable changes are removing the launch event zoom link and moving the event to past events in the homepage. 

Will look into adding that .prettierc file and updaring settings.json, thanks!